compiler/control/OMROptions.cpp

```diff
@@ -3365,15 +3366,15 @@ OMR::Options::processOptionSet(
          //filter starts with a limitfile linenumber range
          else if (*options == '[')
             {
-            filterHeader = options;
+            filterHeader = (char *)options;
```

Comment:
`filterHeader` and `options` are both `const char *`, so this cast shouldn't be needed.